api/node/routes.go

```diff
@@ -134,7 +160,20 @@ func Statistics(c *gin.Context) {
 
 // StatusMetrics returns the node statistics exported by an StatusMetricsHandler without p2p statistics
 func StatusMetrics(c *gin.Context) {
-	ef, ok := c.MustGet("elrondFacade").(FacadeHandler)
+	efObj, ok := c.Get("elrondFacade")
```

Comment:
you can leave a TODO (no need to do this on this PR) to add commom methods like `RespondWithNilAppContext` or `RespondWithInvalidAppContext`

Reply:
The best TODO you can write is actually the one you don't write. 
Added common function in each package as otherwise would have been an interface-usage nightmare. :)